docs_requirements.txt

```
@@ -7,10 +7,9 @@ coverage==3.6
 distribute==0.6.34
 docutils==0.10
 invoke==0.2.0
-mock==1.0.1
 pexpect-u==2.5.1
 py==1.10.0
-pytest==2.3.4
+pytest==6.2.5
```

Bumping this just to see if that runs the tests.

```suggestion
pytest==6.2.6
```